### DISCUSSION TITLE
Request to stop requiring uppercase characters in passwords

### DISCUSSION THREAD

**Discussion by LoganDark** [rustdesk/rustdesk#10179] [General]
I have a password over 20 chars long that has lowercase, digits, and symbols, but I can't use it because RustDesk decides it wants uppercase instead.

This makes zero sense for security considering the password I'm trying to use...

Can we please stop requiring uppercase characters in passwords, or at least replace it with something like "uppercase or symbol"?

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/2888
